ipatests/test_integration/test_sso.py

```diff
@@ -53,6 +54,40 @@ def keycloak_login(host, username, password, username_fl=None):
         host.run_command(["rm", "-f", "/tmp/keycloak_login.py"])
 
 
+def keycloak_add_user(host, kcadm_pass, username, password=None,
+                      username_fl=None):
+    if username_fl is None:
+        username_fl = username
+    domain = host.domain.name
+    kcadmin_sh = "/opt/keycloak/bin/kcadm.sh"
+    kcadmin = [kcadmin_sh, "config", "credentials", "--server",
+               f"https://{host.hostname}:8443/auth/",
+               "--realm", "master", "--user", "admin",
+               "--password", kcadm_pass]
+
+    tasks.run_repeatedly(host, kcadmin, timeout=60)
```

Comment:
I assume run_repeatedly to handle when the remote server is not yet up and running?

Reply:
OK, up to you. It beats using sleep()